project/build.sbt

```diff
@@ -1,13 +1,11 @@
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.1")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
 
-addSbtPlugin("com.lucidchart" % "sbt-cross" % "3.0")
+addSbtPlugin("com.lucidchart" % "sbt-cross" % "4.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.3.9")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.2")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.1.6")
```

Comment:
I have no idea why we had this.

Reply:
Looks like we must have added it, because, our current install process uses deb packages published to bintray

Reply:
oh, maybe we just need to update this to a newer version that supports sbt 1.0

Reply:
we're going to build debs differently now, so I ended up removing this.